wordpress-mobile/WordPress-iOS
# Updated several pods and locked more to a specific version

Fixes #7569 

As part of our release process we try to update the CocoaPods dependencies once in a while. I noticed several pods with version numbers not fixed at a specific version number. This corrects that as much as possible.

To test:
1. Clean, build, and run the unit tests.
2. Make sure notifications sync still works while in the app (StarScream updated).

StarScream was the only major pod revision update with this (look at the lock file).

Needs review: @elibud, @koke, @jleandroperez, @nheagy
